test/e2e/satellite_e2e_test.go

```
@@ -0,0 +1,313 @@
+		WithEnvVariable("POSTGRES_DB", "groundcontrol").
+		WithExposedPort(5432).
+		AsService().Start(ctx)
+
+	if err != nil {
+		log.Fatalf("Failed to start PostgreSQL service: %v", err)
+	}
+	return db
+}
+
+func startGroundControl(ctx context.Context, client *dagger.Client, db *dagger.Service) *dagger.Service {
+	projectRoot, err := getProjectRoot()
```

golangci-lint (1.64.8)

ineffectual assignment to err

(ineffassign)